reference/orders.v2.oas2.yml

```
@@ -182,6 +182,15 @@ paths:
                         value: 12
                       price_inc_tax: 12.45
                       price_ex_tax: 10.12
+              Updating an Order's Gift Certificate Consignment:
```

```suggestion
              Updating an Orderʼs Gift Certificate Consignment:
```
otherwise we'll run into linting errors going forward